### ISSUE TITLE
Browser JS error when starting agent

### ISSUE THREAD

**Issue by jeffmaury** [eclipse-che/che#1462]
I'm running the assembly from the openshift plugin in a VirtualBox plugin. When I open the workspace, my browser (Chrome 51.0 / Win10), the JS error dialog is displayed:

![image](https://cloud.githubusercontent.com/assets/695993/15890390/8fd827c8-2d6f-11e6-91b9-c5499cf4bd74.png)

and logs in the container says the client has aborted the connection

**Comment by ghost** [comment]
@jeffmaury How do you open a workspace? Is everything ok on the dashboard when you run it?

Can you provide server and ws-agent logs?

**Comment by james10174** [comment]
@jeffmaury I usually get this error from port blocking issues. Are you running default che.bat or has it been modified? Are you using your local machine running che.bat chrome browser or a different machine?

**Comment by james10174** [comment]
Just got the same error myself after opening IDE, going to dashboard for about 10 minutes, and re-entering workspace again. Not sure if it is related to your error.

I stopped workspace, started workspace again, opened workspace and error went away. I went to dashboard again and then back to workspace but I did not get the error again. I will see when/what is causing this.

![image](https://cloud.githubusercontent.com/assets/16562450/15898731/fb166678-2d5e-11e6-88bd-5593dd763e0e.png)
